src/lib/commits.js

```diff
@@ -0,0 +1,23 @@
+import { exec } from 'child_process'
+
+import { efh } from './error'
```

Comment:
Oh nice to know thanks. I'll backport this then after I'm done with the tests.
Argh this PR is so huge already :grimacing: